src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
You do not need to use inheritance from the ColorSupplier class because the Lottery class performs different logic and only uses the ColorSupplier to get the color. The color is obtained using a static method call.
You just need to remove the inheritance.